tests/Substrait.Lineage.Tests/LineageExtractorVisitorTest.cs

```diff
@@ -0,0 +1,19 @@
+using Substrait.Protobuf;
+
+namespace Substrait.Lineage.Tests
+{
+  [TestClass]
+  public class LineageExtractorVisitorTest
+  {
+    [TestMethod]
+    public void TestMethod1()
+    {
+      Plan plan;
+      using var planPb = File.OpenRead(@"TestData\tpcds\Q02.pb");
```

Comment:
how was the `.pb` file generated? Can you please add instructions somewhere?

Reply:
the `.pb` was generated using `substrait-java:isthmus` ([here](https://github.com/substrait-io/substrait-java/tree/main/isthmus)). Its landing page has the instructions for generating `Substrait` plans using `calcite`. I hope this helps.